core/src/main/java/com/linecorp/armeria/common/logging/BuiltInProperty.java

```diff
@@ -55,6 +55,15 @@
  * @see RequestContextExporterBuilder#builtIn(BuiltInProperty)
  */
 public enum BuiltInProperty {
+    /**
+     * {@code "authenticated.user"} - the authenticated user if exists.
+     */
+    AUTHENTICATED_USER("authenticated.user", log -> {
+        if (log.isAvailable(RequestLogProperty.AUTHENTICATED_USER)) {
+            return log.authenticatedUser();
+        }
+        return null;
+    }),
```

Comment:
Nit: we can use a ternary operator for consistent with others, something like this:
```
AUTHENTICATED_USER("authenticated.user", log -> 
    log.isAvailable(RequestLogProperty.AUTHENTICATED_USER) ? log.authenticatedUser() : null
),
```